src/quacc/recipes/espresso/_base.py

```diff
@@ -209,7 +231,7 @@ def _prepare_atoms(
 
     calc_flags = recursive_dict_merge(calc_defaults, calc_swaps)
 
-    atoms.calc = Espresso(
+    calc = Espresso(
```

Comment:
Not sure there was a need to change it from `atoms.calc = ` but it doesn't matter much I suppose.